### ISSUE TITLE
PMS Service Does not Restart Plex

### ISSUE THREAD

**Issue by LitlJayHeise** [Mart11/PlexUpdate4Win#4]
The update script(s) work great.  I have it running at 8am and creating a log entry on the desktop, so I can always go in and check on it, and I am receiving emails each time it installs an update.  It is not restarting Plex after an update, though.  cjmurph's plex service is definitely running after the update, but if I log into the server and right click on his tray icon, it lists the option to start plex.  I can do it manually and things work fine after that, but the whole idea of automating the update is so that I won't have to do that.  Could something be borked in my config, or is this a known issue?

**Comment by LitlJayHeise** [comment]
FWIW, I got the service to start Plex again by adding 2 lines at the end of the sendmail script to stop and start plexservice (I suppose I could just as easily add them to PlexUpdate4Win.cmd).  This seems like a pretty inelegant solution to me, but it works.  I'd love to know if there is a better way.

**Comment by Mart11** [comment]
After a new version is installed by update script(s), the PmsService will restart the Plex Media Server automatically after 300 seconds by default. This 300 seconds should be more that enough for installation process - or you can adjust it if needed in settings. Check if you have the option "Automatically Restart Plex if it Stops" selected in settings (through systray icon).

**Comment by LitlJayHeise** [comment]
I see the the option was un-checked in my settings.  It is completely possible that I turned it off while working on the system some time in years past so that it wouldn't restart and cause an issue while I was under the hood.  This machine has lots of horsepower and should be able to complete a silent install in 30-60 seconds, so I set the timeout to 180.  For the time being I have commented out the lines I added and will wait for the next update to see how it goes.  Thanks!
